main.go

```diff
@@ -46,6 +46,12 @@ func main() {
 	e.DELETE("/waypoint/:waypointId", controllers.DeleteWaypoint)
 	e.GET("/waypoints", controllers.GetAllWaypoints)
 
+	//Ground Object
+	e.POST("/ground_object", controllers.CreateGroundObject)
+
```

Comment:
Methods to edit, update and delete GroundObject's also need to be added

Reply:
Added the methods but missing some documentation since I'm not too sure how to write it

Reply:
Added documentation.